scripts/sanitise-db.sql

```diff
@@ -8,19 +8,24 @@ DECLARE
     query TEXT;
     count_query TEXT;
     pre_update_count INTEGER;
+    table_specific_condition TEXT := '';
 BEGIN
     RAISE NOTICE 'Anonymising table: %, data_blob key: %, with lorem ipsum.', UPPER(table_name), UPPER(data_blob_key);
 
+    IF table_name = 'project_progress' THEN
```

Comment:
What is the benefit of deliberately not sanitising `project_progress` records where the start date is less than the end date?

Reply:
Our code checks the constraint making sure that the start_date must always be before the end_date. When sanitising project_progress,I encounter some errors for entries that violate this constraint, so we only update records where this condition is met.

Reply:
Ah cool I've looked into this and the reason is that even though we've added a database constraint for start date and end date to the `project_progress` table, existing rows were not validated against that constraint at migration time. When you run the `UPDATE` statement in this sanitisation script against a violating row, the constraint is re-validated and the query fails

From checking the prod database just now, only one row fails that constraint. We ran a re-ingest last week to cover the other cases of this case failing, but this row remains outstanding. @samuelhwilliams can we update this final violating row in prod - whether manually or through reingest - to avoid Israr having to add this condition into the sanitisation script?

Reply:
have asked about that last bit of data again...